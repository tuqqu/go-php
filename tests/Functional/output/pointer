1
1
0
true 6 6
[1 12 3]
[1 12 13]
