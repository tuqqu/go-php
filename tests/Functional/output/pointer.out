1
1
0
true 6 6
[1 12 3]
[1 12 13]
some_string string2
